service/pixelated/resources/__init__.py

```diff
@@ -122,6 +122,9 @@ def feedback_service(self, request):
     def soledad(self, request):
         return self._service(request, '_leap_session').soledad
 
+    def get_backup_email(self, request):
+        return json.loads(request.content.getvalue()).get('backupEmail')
```

Comment:
Shouldn't this be only inside the BackupAccountResource? It only exists in that resource, so this will break for all other resources.